### PR TITLE
AzureStack Gallery Admin

### PR DESCRIPTION
**Pull request by deathly809**
Initial commit for the AzureStack Gallery admin specs.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by bganapa — APPROVED** [review]

**Comment by deathly809** [comment]
@lmazuel Hmm, looks like it does not like objects with no properties.  Not really sure what to do with this.

**Comment by deathly809** [comment]
@lmazuel Ping.

File: specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preivew/2015-04-01/Gallery.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                "tags": [
+                    "Operations"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "Gallery.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "summary": "Gets the available gallery admin operations.",
+                "operationId": "ListOperations",
```
Comment (jhendrixMSFT) [inline specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preivew/2015-04-01/Gallery.json]: This really should be Operations_List.

File: specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preivew/2015-04-01/Gallery.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/OperationList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
```
Comment (jhendrixMSFT) [inline specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preivew/2015-04-01/Gallery.json]: Curious why this is null?  There's a nextLink field defined in OperationList.

File: specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preivew/2015-04-01/GalleryItem.json [hunk 3]
```diff
@@ -0,0 +1,700 @@
...
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/GalleryItemList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
```
Comment (jhendrixMSFT) [inline specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preivew/2015-04-01/GalleryItem.json]: Another null nextLinkName.

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**